src/modules/auth/actions/load-account-history.js

```diff
@@ -25,78 +26,56 @@ export const loadAccountHistory = (loadAllHistory, triggerTransactionsExport) =>
     dispatch(syncUniverse())
   }
 
-  if (registerBlock && oldestLoadedBlock && oldestLoadedBlock !== registerBlock) {
-    const options = {}
-    if (!loadAllHistory) {
-      options.toBlock = oldestLoadedBlock === blockchain.currentBlockNumber ? oldestLoadedBlock : oldestLoadedBlock - 1
+  // TODO: figure out where this data comes from and if it is needed
```

Comment:
They’re not explicitly needed any longer — they’re a carry over from the old chunked loading system from on-chain.